masterlist.yaml

```diff
@@ -824,3 +824,29 @@ plugins:
       - name: 'master_index.esp'
         display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
         condition: 'not active("MasterIndexRedux.esp")'
+
+  - name: 'The_Vanilla_Quest_Tweaks_RP_Choices_Consequences_Super_Mega_Package_-_Ultimate_Edition.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/47466/'
+        name: 'The Vanilla Quest Tweaks RP Choices Consequences Super Mega Package - Ultimate Edition'
+    inc:
+      - 'A Cure for Vampirism - Skink''s Solution.esp'
+      - 'Champion of Clutter.esp'
+      - 'Duel of Honor - Improve the Chances.esp'
+      - 'Fargoth in Distress.esp'
+      - 'Fort Moonmoth Fundraiser Dinner.esp'
+      - 'Harvest''s End Festival.esp'
+      - 'Hentus Needs Pants Overhaul.esp'
+      - 'Hiring Guards for the Redoran Stronghold - Honorable Solution.esp'
+      - 'Immersive Neloth Reward.esp'
+      - 'Libertarian Magical Services.esp'
+      - 'Morrowind - Immanuel Kant Edition.esp'
+      - 'Oath to Saint Roris Instead.esp'
+      - 'Pacifist Options - When it Makes Sense.esp'
+      - 'Redoran Founder''s Helm Add-on - Honor the Ancestors.esp'
+      - 'Strange Man at Gindrala Hleran''s House Overhaul.esp'
+      - 'Teach Nels Llendo a Lesson.esp'
+      - 'The Corpse and the Skooma Pipe Overhaul.esp'
+      - 'The Frostmoth Smugglers - Properly Rewarded.esp'
+      - 'Thelas'' Pillows Overhaul.esp'
+      - 'Therana vs. Trerayna.esp'
```

Comment:
The file extensions can be capitalized to match the files as they're provided (it doesn't make a difference either way, it's just for consistency. I also noticed that `Redoran Founder's Helm Add-on - Honor the Ancestors.esp` is actually `Redoran Founder's Helm Add On- Honor the Ancestors.ESP` (i.e., `Add-on -` -> `Add On-`) & `Therana vs. Trerayna.esp` is actually `Therana vs Trerayna.ESP` (i.e., `vs.` to `vs`):

```suggestion
      - 'A Cure for Vampirism - Skink''s Solution.ESP'
      - 'Champion of Clutter.ESP'
      - 'Duel of Honor - Improve the Chances.ESP'
      - 'Fargoth in Distress.ESP'
      - 'Fort Moonmoth Fundraiser Dinner.ESP'
      - 'Harvest''s End Festival.ESP'
      - 'Hentus Needs Pants Overhaul.ESP'
      - 'Hiring Guards for the Redoran Stronghold - Honorable Solution.ESP'
      - 'Immersive Neloth Reward.ESP'
      - 'Libertarian Magical Services.ESP'
      - 'Morrowind - Immanuel Kant Edition.ESP'
      - 'Oath to Saint Roris Instead.ESP'
      - 'Pacifist Options - When it Makes Sense.ESP'
      - 'Redoran Founder''s Helm Add On- Honor the Ancestors.ESP'
      - 'Strange Man at Gindrala Hleran''s House Overhaul.ESP'
      - 'Teach Nels Llendo a Lesson.ESP'
      - 'The Corpse and the Skooma Pipe Overhaul.ESP'
      - 'The Frostmoth Smugglers - Properly Rewarded.ESP'
      - 'Thelas'' Pillows Overhaul.ESP'
      - 'Therana vs Trerayna.ESP'
```